NOTICE

```diff
@@ -2,10 +2,16 @@ xayn_dart_api_dl
 Copyright 2021 Xayn AG
 
 This product includes software developed at
-Xayn AG (https://www.xayn.com/company/contact).
+Xayn AG (https://www.xayn.com).
 
-This product bundles parts of the dart language sdk
-(https://github.com/dart-lang/sdk/) for the purpose of
-interfacing with the dart language, which is available
-under a "3-clause BSD" license. For details, see
-dart-src/.
\ No newline at end of file
+Except as otherwise noted (below and/or in individual files), this project is
+licensed under the Apache License, Version 2.0 <LICENSE>.
+
+The xayn_dart_api_dl project includes packages written by third parties.
+The following third party packages are included, and carry
+their own copyright notices and license terms:
```

Comment:
`packages`  isn't really the right terminology here, I think something like "bundles code written by third parties" is a better choice of wording.

Reply:
The problem is `package` is a word with a lot of contexts depending on where it appears and by whom it's read.